cors/cors_test.go

```diff
@@ -0,0 +1,354 @@
+package cors
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/mount-joy/thelist-lambda/handlers/testhelpers"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_keysToString(t *testing.T) {
+	tests := []struct {
+		name                   string
+		input                  map[string]bool
+		expectOutputToContain  []string
+		expectedNUmberOfCommas int
+	}{
+		{
+			name:                   "empty map returns empty string",
+			input:                  nil,
+			expectOutputToContain:  nil,
+			expectedNUmberOfCommas: 0,
+		},
+		{
+			name:                   "add key from map to string in output",
+			input:                  map[string]bool{"hello": true},
+			expectOutputToContain:  []string{"hello"},
+			expectedNUmberOfCommas: 0,
```

Comment:
the number of commas check doesn't really work too well to distinguish the first two, since they're both the same, but maybe it could assert on the string length?